applications/StructuralMechanicsApplication/tests/quadratic_test/static_quadratic_quad_test.mdpa

```diff
@@ -0,0 +1,183 @@
+Begin ModelPartData
```

Comment:
would it be possible to hard code this in the python file (with createnewnode etc...)? this is not a strict request, however it would be one file less